### ISSUE TITLE
Look up OIDC discovery document

### ISSUE THREAD

**Issue by nbarbettini** [nbarbettini/oidc-debugger#24]
Allow the user to paste the URL to an authority, and inspect the discovery document to make suggestions for things like Authorize URI, scopes, response modes and types, etc.

**Comment by jerriep** [comment]
How do you envision the UX for this to work?

**Comment by nbarbettini** [comment]
My initial thought: if you paste a URL into the Authorize URI field that can be fetched as a discovery document, update the UI to:

* Indicate a discovery document was loaded
* Include a link to view the raw discovery document
* Auto-populate Authorize URI
* Provide suggestions/guards around supported grant types

**Comment by nbarbettini** [comment]
However, it's probably mixing things up (and would be confusing) to use the Authorize URI field for this. Maybe another field or small button above Authorize URI that would let you paste this in.

As you can tell I haven't fully designed this in my head yet. 😄 Feel free to suggest your own ideas!

**Comment by jerriep** [comment]
Yeah, I agree mixing things up will lead to confusion, and having a specific action (like a button) + additional UI for this would make most sense

**Comment by CrowdHailer** [comment]
Can I help in anyway on this Issue. It would be really useful for me. 

From a UX point of view I would expect something like

```
Enter issuer url here
[input]
[Button fetch configuration]

Some Break

Or manually configure below.
```

Once the discovery document is fetch it can just auto fill the same fields that were just being used for manual configuration.

There is another debugger from Auth0 that does something fairly similar. https://openidconnect.net/

![screenshot](https://user-images.githubusercontent.com/5862302/74602901-73274200-50a5-11ea-9d13-df8714fbcfe4.png)

**Comment by nbarbettini** [comment]
@CrowdHailer I think this could work entirely on the client side (`fetch()` the discovery document with JavaScript). If you have time to build it in JS, I'm happy to accept PRs.

**Comment by CrowdHailer** [comment]
I might be able to have a crack at this later on next week.
